test/extensions/tracers/xray/tracer_test.cc

```diff
@@ -347,6 +347,7 @@ TEST_F(XRayTracerTest, ChildSpanHasParentInfo) {
     // Hex encoded 64 bit identifier
     EXPECT_STREQ("00000000000003e7", s.parent_id().c_str());
     EXPECT_EQ(expected_->span_name, s.name().c_str());
+    EXPECT_EQ(Subsegment, s.type().c_str());
```

Comment:
can we update the test to check that `type()` is not set for a parent span?

Reply:
done